src/grisp_onewire.erl

```diff
@@ -131,8 +132,8 @@ handle_call({transaction, Fun}, _From, State) ->
     Reply = try
         {result, Fun()} % TODO: Implement timeout for transactions
     catch
-        Class:Reason ->
-            {exception, Class, Reason, erlang:get_stacktrace()}
+	?EXCEPTION(Class, Reason, StackToken) ->
```

Comment:
Looks good! Could you please update the variable name to `Stacktrace` here and below?